packages/opentelemetry-core/README.md

```diff
@@ -1,3 +1,16 @@
 # OpenTelemetry Core
 
-TODO
+This package provides concrete implementations of the OpenTelemetry types for trace and metrics. It's intended for use both on the server and in the browser.
```

Comment:
Instead of concrete, I would say default and no-op implementations. WDYT?

Reply:
"concrete" was more "generic", but you're right for what this package is doing for now. I think when the package will do more, we will evolve the readme too! 👍 Thanks for the feedback.